tests/robot-cases/Group1-Nightly/Common.robot

```diff
@@ -693,3 +693,21 @@ Test Case - Read Only Mode
     Push image  ${ip}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}  project${d}  busybox:latest
     Close Browser
 
+Test Case - Proxy Cache
+    ${d}=  Get Current Date    result_format=%m%s
+	${registry}=  Set Variable  https://hub.docker.com/
+    Init Chrome Driver
+    Sign In Harbor  ${HARBOR_URL}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}
+	Switch To Registries
+	Create A New Endpoint  docker-hub  e1${d}  ${registry}  danfengliu    Aa123456
+    Create An New Project And Go Into Project  project${d}  proxy_cache=${true}  registry=e1${d}

```

Comment:
Could you please add a testcase to pull image through proxy cache.

Reply:
OK, Got it.